types/parse5/index.d.ts

```diff
@@ -37,6 +40,8 @@ export interface Location {
     startLine: number;
 }
 
+export interface EndLocation extends EndLocationBase, Partial<ElementLocationBase> {}
```

Comment:
this shouldn't be `Partial` (an end location always has an `endTag`)

Reply:
i assumed this is the case from our typescript port we've been working on, but actually its wrong there too! good catch. it should remain partial

Reply:
The `endTag` is absent for void tags and auto‑closed `<p>`s.